docs/user-guide/content-video-toolbar.md

```diff
@@ -0,0 +1,23 @@
+# Video Content Toolbar
+**********************
+
+As soon as a video content is added, the Video Content Toolbar appears on top of the video: 
+
+<img src="../img/content-video-toolbar/c-video-toolbar.jpg" class="ms-docimage"/>
+
+Through this toolbar, the story editor is able to perform the following operation:
+
+* **Change media source** <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> accessing the [Media Editor](media-editor-window.md)
+
+* **Mute video** <img src="../img/button/mute-video-button.jpg" class="ms-docbutton"/> to disable the video audio
+
+* **Enable Autoplay** <img src="../img/button/autoplay-button.jpg" class="ms-docbutton"/> to play the video automatically once the user is on it
+
+* **Enable Loop** <img src="../img/button/loop-botton.jpg" class="ms-docbutton"/> to continuously repeat the video
+
+* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the image. (It is available if the description is added to the image on the [Media Editor](media-editor-window.md))
+
+* **Remove** <img src="../img/button/remove2.jpg" class="ms-docbutton"/> the video content
+
+!!! note
+    The video play based on the selected options only in *View Mode*.
```

Comment:
Inside the Media Editor you can watch a preview of the video before adding it to the story. The video play will be available only in *View Mode* of the story: it is not available in edit mode except in the media editor as a preview.